ext/opcache/zend_shared_alloc.c

```diff
@@ -541,18 +541,18 @@ void zend_shared_alloc_restore_xlat_table(uint32_t checkpoint)
 	zend_hash_discard(&ZCG(xlat_table), checkpoint);
 }
 
-void zend_shared_alloc_register_xlat_entry(const void *old, const void *new)
+void zend_shared_alloc_register_xlat_entry(const void *key_pointer, const void *value)
```

Comment:
nit: a little inconsistency with the header and here, ie `key_pointer` vs `key`, I guess you re aiming towards the latter.

Reply:
ok just detail indeed, looks ok otherwise cc @Girgias